package.json

```
@@ -24,7 +24,7 @@
     "github-from-package": "0.0.0",
     "minimist": "^1.2.3",
     "mkdirp-classic": "^0.5.3",
-    "napi-build-utils": "^1.0.1",
+    "napi-build-utils": "^2.0.0 || ^1.0.1",
```

```suggestion
    "napi-build-utils": "^2.0.0",
```

Otherwise we would end up with the buggy version if some other package in the `node_modules` tree depends on `napi-build-utils@1`.

---

That works too.  Fixed!